### PR TITLE
Pokemon Card and Grid

### PR DESCRIPTION
**Pull request by Emmanuelcazlo**
# Pull Request

### Ticket impemented #[POK-19](https://coucat.atlassian.net/browse/POK-19)

[//]:<> (You can see the number of ticket in Jira in the top of task, replace the ticketNumber in the space and the number in the link)
### What did you implement it?
 
[//]:<> (Mark with an 'x' or a space)

- [ ] Bug Fix (Non-breaking change wich fixes an issue)
- [x] New Feature (Non-breaking change wich adds functionality)
- [ ] Breaking Change (Fix or feature that would cause existing functionality to not work as expected)

### How did you implement it?
[//]:<> (In this section describe all things that you made)

The pokemon card was made follow the figma design with a little changes in some sizes of.
Add prop-types to use props in the pokemon card and set all things that we need like name, number or id, image and types or something that we need in the future.
Set a hover animation of up and down in the container.

## How can we test this ticket?
[//]:<> (Write in this part all the commands that you can use to test your code)
To see the hover animation you only need put the mouse over the container.
To test the props you can add a new pokemonCard in /components/pokemonCard/index.jsx and set the props as in the existing pokemonCard, also you can change the types in the same page in the object **types**. You can see all types in pokemonCard/styles.jsx. To see other image you only need change the number in the end of the link like:
 ```sh
https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/versions/generation-v/black-white/animated/{any-number}.gif
 ```
## Checklist
[//]:<> (Mark with a 'x' or a space )

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by Emmanuelcazlo** [comment]
In the index.jsx of pokemonCard i'm add a types obj to test the automatic type set with props

File: src/components/pokemonCard/styles.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  width: 210px;
+  height: 150px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  background-color: white;
+  border-radius: 10px;
+  margin: 20px;
```
Comment (pablo179) [inline src/components/pokemonCard/styles.jsx]:
```suggestion
  margin: 20px;
```
```suggestion
  margin: 20px 10px;
```

File: src/components/pokemonCard/styles.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    width: 70px;
+    height: 60px;
+    position: absolute;
+    top: -30px;
+    object-fit: contain;
+`;
+export const Number = styled.span` 
+    font-size: 15px;
+    color: #8F9396;
+    margin-top: 10px;
```
Comment (pablo179) [inline src/components/pokemonCard/styles.jsx]:
```suggestion
    margin-top: 5px;
```

**Review by pablo179 — COMMENTED** [review]
good implementatin of the card, just some little changes on styles

**Review by pablo179 — APPROVED** [review]
LGTM

**Pull request merged**